backend/src/main/java/com/htecgroup/skynest/service/impl/UserServiceImpl.java

```diff
@@ -186,12 +186,33 @@ public UserDto verifyUser(UserDto userDto) {
     return userDto.withEnabled(true).withVerified(true);
   }
 
-  public void authorizeAccessToUserDetailsWith(UUID uuid) {
+  public void authorizeViewUserDetailsWith(UUID uuid) {
+    LoggedUserDto loggedUserDto = currentUserService.getLoggedUser();
```

Comment:
Try to rewrite this with custom annotation instead. Applies for all new methods. Also try to create the methods reusable, this way you have lots of duplicate code